Application/Services/PostService.cs

```diff
@@ -0,0 +1,131 @@
+﻿using Application.Dtos.Blog;
+using Application.Exceptions;
+using Application.Interfaces;
+using AutoMapper;
+using Core.Constants;
+using Core.Entities;
+using Core.Repositories;
+using Mapster;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.EntityFrameworkCore;
+using System.ComponentModel.DataAnnotations;
+
+namespace Application.Services
+{
+    public class PostService : IPostService
+    {
+        private readonly IHttpContextAccessor _httpContextAccessor;
+        private readonly UserManager<User> _userManager;
+        private readonly IPostRepository _postRepository;
+        private readonly IMapper _mapper;
+
+        public PostService(IPostRepository postRepository, IHttpContextAccessor httpContextAccessor,
+            UserManager<User> userManager, IMapper mapper)
+        {
```

Comment:
```suggestion
        public PostService(
        IPostRepository postRepository,
        IHttpContextAccessor httpContextAccessor,
        UserManager<User> userManager,
        IMapper mapper
)
        {
```
keep all your dependencies in the constructor on their separate line. That way you will be consistent and have a clear view at first sight.